[RM Ch09] Automation section moved to chapter end (#3220)
cntt-n/CNTT: doc/ref_model/chapters/chapter09.rst

Configuration and Lifecycle Management section deals with the changes being made to the infrastructure hardware,
software, and management components (e.g. changing of number of hypervisor hosts from 10 to 12).

Telemetry and Observability
---------------------------

> Operating complex distributed systems, such as a Telco network, is a demanding and challenging task that is continuously

Reviewer: 1. "demanding and challenging" are synonyms
2. task or job (consists of multiple tasks)

Author: @pgoyal01, please check the new wording

> being increased as the network complexity and the production excellence requirements grow. There are multiple reasons

Reviewer: "... continually being increased ..." sounds very odd. since "complexity is used later in the sentence, may I suggest (not the best) to replace "being increased" with "becoming more so"

Author: @pgoyal01, please check the new wording

> being increased as the network complexity and the production excellence requirements grow. There are multiple reasons

Reviewer: "... production excellence" can mean different things to different folks.
What does "production excellence" look like in the cloud era or Kubernetes? Both recognise failure.

Author: @pgoyal01, please check the new wording

why it is so, but they originate in the nature of the system concept. To reach the ability of providing Telco services,
a complex system is decomposed into multiple different functional blocks, called network functions. Internal
communication between the diverse network functions of a distributed system is based on message exchange. To formalize
this communication, clearly defined interfaces are introduced, and protocols designed. Even though the architecture of
a Telco network is systematically formalized on the worldwide level, heterogeneity of services, functions, interfaces,
and protocols cannot be avoided. By adding the multi-vendor approach in implementation of Telco networks, the outcome is

> indeed a system with remarkably high level of complexity which requires significant efforts for managing and operating

Reviewer: ```suggestion
a system with a remarkably high level of complexity that requires significant effort for managing and operating
```

it.

> To ensure proper support and flawless work in the large ecosystem of end user services, a formalized approach directed

Reviewer: ```suggestion
A large and complex ecosystem of end-user services requires a formalized approach
```

> towards high reliability and scalability of systems is required. The discipline which applies well known practices of

Reviewer: ```suggestion
for achieving high reliability and scalability. The discipline which applies well-known practices of
```

software engineering to operations is called Site Reliability Engineering. It was conceived at Google, as a means to
overcome limitations of the common DevOps approach.

> Common supporting system (OSS – Operation Support System, BSS – Business Support System) requirements are redefined,

Reviewer: ```suggestion
Common support system (such as, OSS – Operation Support System, BSS – Business Support System) requirements are redefined,
```
